data/human/pirates/pirate intro.txt

```diff
@@ -0,0 +1,732 @@
+
```

Comment:
You should probably put a copyright header here.

Reply:
No, not a copyright header. Some people make the moral choice to release a work to the public domain. As this is the pirate campaign, releasing without copyright fits the style of the campaign. It is also a valid choice by the author, if they want to do it. However, the lack of a header does *not* release it without copyright.

It needs a CC0 public domain dedication, and the style checker has to be updated to accept CC0. I'll do a PR to this branch shortly to make that update.

You can't *force* someone to claim copyright, nor would it be morally right to do so.

Reply:
@UnorderedSigh I'd let Lake speak for himself before typing out a debate on having copyright or not for the game. It's common practice for all of our content writers. It's okay if you don't like it, but don't do the opposite and try to push someone *away* from doing so. Leave it grey-lined.

Reply:
It appears Lake on discord is going to decline copyright. That is totally fine. In the future Sigh, please do not make these decisions for content that isn't yours, and do not press them in a direction of your own choice. Please let the creator decide for themself.

Reply:
I haven't spoken for anyone. I've just given the suggestion. However, both my sister and I have asked to release in CC0 and been denied, so I'm happy to fight for a fellow pirate.

I made PR to this branch which adds a CC0 dedication to the `pirate intro.txt` with an appropriate header line.

https://github.com/Endless-Sky-Pirate-Campaign/endless-sky/pull/27

Reply:
The whole game is MZ unless specified otherwise anyway, isnt it?

Reply:
The file, on its own, is only (C) MZ if there is a notice in the file saying so.

Reply:
Thanks, Sigh!